Sources/StreamChatUI/Composer/ComposerVC.swift

```diff
@@ -244,6 +247,12 @@ open class ComposerVC: _ViewController,
     override open func setUp() {
         super.setUp()
 
+        let client = channelController?.client
+        if let cid = channelController?.cid {
+            memberListController = client?.memberListController(query: .init(cid: cid))
+            memberListController?.synchronize()
```

Comment:
hope this also doing the job not on a main queue

Reply:
Everything on LLC is not doing work on the main thread ofc